packages/gatsby-transformer-react-docgen/src/__tests__/displayname-handler.js

```diff
@@ -0,0 +1,150 @@
+import test from "ava"
```

Comment:
I believe this fixes jest. It would be nice if you could transform it into a more gatsby like test with `describe` and `it`
```suggestion
```